master/getting-started/calicoctl/configure/etcd.md

```diff
@@ -46,6 +66,33 @@ spec:
   etcdCertFile: /etc/calico/cert.pem
   etcdCACertFile: /etc/calico/ca.pem
 ```
+#### Example configuration file with inline CA certificate, client certificate and key
+
+```yaml
+apiVersion: projectcalico.org/v3
+kind: CalicoAPIConfig
+metadata:
+spec:
+  etcdEndpoints: https://etcd1:2379,https://etcd2:2379,https://etcd3:2379
+  etcdKey: `-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Thanks for including an example but I'm not sure that this is valid YAML, I think a YAML I think you want to use "literal" style, which starts with a `|` character:
```
etcdKey: |
  -----BEGIN RSA PRIVATE KEY-----
  k0dWj16h9P6TvfcNl2iwT4VIwx0uy2faWBED1DrCJcuQCy5nPrts2ZIaAWPi1t3t
  VbDKQvs+KXBEeqh0qYcYkejUXqIF0uKUFLjiQmZssjpL5RHqqWuYKbO87n+Jod1L
  TjGRHdbP0zF2U0LdjM17rc2hpJ3qrmgJ7pOLzbXMcOr+NP1ojRCArXhQ4iLs7D8T
  eHw9QH4luJYtnmk7x03izLMQdLWcKnUbqh/xOVPyazgJHXwRxwNXpMsBVGY=
  -----END RSA PRIVATE KEY-----
```

Reply:
Acknowledged and updated.